### ISSUE TITLE
[Translations Phase 2] [Accessibility] - Add language not announced as button with voice over

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#33547]
### **Steps to Reproduce**

1. Enable voice over from iOS settings.
2. Launch Firefox.
3. Go to settings->translation->add language
4. Listen

### **Expected Behavior**

Add language is announced as button.

### **Actual Behavior**

Add language is NOT announced as button.

### **Device & Build Information**

- Device: iPhone 15 (17.6)
- Build version: v9000 (71228)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-15665)
